### ISSUE TITLE
Host status

### ISSUE THREAD

**Issue by vhostvn** [Janusec/janusec#23]
Hi,

When we acccess Health -> Hosts Status we see **Last Access Time**: 1/1/1970 07:00:00

Is this a mistake? And how to rectify this?

**Comment by zhyale** [comment]
No, it is not a mistake. It means that no request recorded after the gateway started.

**Comment by vhostvn** [comment]
We tested this morning and it said: 

![image](https://user-images.githubusercontent.com/16444776/132783334-37e4a4f9-4e4e-42cf-95dc-721ecebfb389.png)

and a lot like below:

![image](https://user-images.githubusercontent.com/16444776/132783456-e558799c-7131-44e2-a159-99b3c3242416.png)
